src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -172,6 +162,13 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
             CancellationToken cancellationToken)
         {
             var implementation = classOrStructType.FindImplementationForInterfaceMember(member);
+
+            if (implementation?.ContainingType.TypeKind == TypeKind.Interface)
+            {
+                // Treat all implementations in interfaces as explicit, even the original declaration with implementation.
```

Comment:
this could benefit from a 'why' explanation.  Perhaps: this way we don't regenerate unnecessary impls when the declaration already has a perfectly good impl that can be used.